shared/chat/conversation/info-panel/notifications/container.js

```diff
@@ -98,6 +108,5 @@ export default compose(
       this.props._resetNotificationSaveState()
     },
   }),
-  // $FlowIssue doesn't like dynamic props like we do above
-  branch(props => !props.conversationIDKey, renderNothing)
+  branch(props => !('conversationIDKey' in props), renderNothing)
```

Comment:
ah I've never seen this syntax before. the change fixes the `$FlowIssue` I'm guessing?

Reply:
we shouldn't be using branch at all for these types of things

Reply:
it also looks like `'conversationIDKey' in props` will be true if  
```js
props = {conversationIDKey: undefined}
```
which might be bad

Reply:
Reverted this. Agree that we want to get rid of branch entirely, but I didn't want to make this PR bigger. Will do in a separate PR.